feature/experimental/gribi/ate_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2022 Google LLC
```

Comment:
I think I have enough confidence in this test that we can put it in the canonical location: `feature/gribi/ate_tests/ack_in_presence_other_routes/...`

https://github.com/openconfig/featureprofiles/tree/main/feature/gribi/ate_tests

Thank you for the contribution!

Reply:
Thank you for the review.